interceptors/logging/payload.go

```diff
@@ -28,6 +29,9 @@ func (c *serverPayloadReporter) PostMsgSend(req interface{}, err error, duration
 	if err != nil {
 		return
 	}
+	if c.decision != LogResponse && c.decision != LogRequestAndResponse {
```

Comment:
Can we use `switch case` in case we want to add more conditionals later on for different `PayloadDecision`? Would be easy to read and maintain the code 😄

Reply:
Yes sure, good idea. I just changed it.